telehealth-example/src/main/assets/memberships.json

```diff
@@ -0,0 +1,18 @@
+[
```

Comment:
Why it's stored in `assets`, when the rest of `json` files are stored inside `raw` directory?

Reply:
Done